notebooks/post-startup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Default post startup script for AI notebooks.
+# AI Notebook post startup scrips are run only when the instance is first created.
+
+# Send stdout and stderr from this script to a tmp file for debugging.
+exec >> /tmp/post-startup-output.txt
+exec 2>&1
+set -x
+
+# The linux user that JupyterLab will be running as. It's important to do some parts of setup in the
+# user space.
+JUPYTER_USER="jupyter"
+
+# Install common packages in conda environment
+/opt/conda/bin/conda install -y pre-commit nbdime nbstripout pylint pytest
```

Comment:
looking forward to seeing in a future PR how you would like to package up the setup needed for specific tutorial notebooks (e.g. there are a default set of tutorials we install, and the setup script for those tutorials first calls this setup script)

Reply:
we could make a read-only deploy key for the repo and store it in the metadata